### ISSUE TITLE
Markdown formatting is adding line breaks around 80 characters, Prettier does not add any by default.

### ISSUE THREAD

**Issue by David-Else** [denoland/deno#9191]
@dsherret  In the following example.
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
`prettier --write  filename.md` does not change the formatting. `deno fmt filename.md` creates:
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
incididunt ut labore et dolore magna aliqua.
```
An unwanted linebreak has been inserted, this makes it impossible to work with any text that needs to reflow... like 99% of text on the web. Thanks :)

**Comment by kitsonk** [comment]
Markdown is supposed to reflow itself.  If you take a look at our documentation, it is all formatted with dprint at 80 cols and reflows perfectly fine when rendered as HTML.  Markdown requires two line breaks to actually break a line.

**Comment by David-Else** [comment]
@kitsonk OK, that is interesting. I had not actually tested it with HTML, it was just a shock to see the non-standard formatting. I think it would be much better to leave it at the Prettier default as the editor can take care of the word wrapping.

VSCode soft wraps with:
```
"editor.wordWrap": "wordWrapColumn",
```
and Vim/Neovim soft wraps to Window size with:
```
  " wrap at word boundaries rather than right at the terminal edge
  set linebreak
```
Having it hard wrapped at 80 cols can be super annoying, and differs from the expected Prettier default compatability.

It would be interseting to hear what other people think.

**Comment by dsherret** [comment]
Just a note that it's configured to do this here, which is similar to prettier's "pros-wrap: always` config:

https://github.com/denoland/deno/blob/ffb4b32e9246dec2d452f81d530e8738e19e5a4e/cli/tools/fmt.rs#L287

**Comment by kitsonk** [comment]
> it was just a shock to see the non-standard formatting

I disagree that it is non-standard... It just doesn't meet your expectations.

> Having it hard wrapped at 80 cols can be super annoying.

Deno has been consistent in hard wrapping at 80 cols, be it code, or be it now markdown.

@dsherret thanks for the heads up on the config.

**Comment by wojpawlik** [comment]
While I don't mind hard-wrapping long lines, joining shorter lines causes noisy diffs:

![obraz](https://user-images.githubusercontent.com/23058303/116445173-d853f700-a855-11eb-8ad1-e4e3b01ba45a.png)

Left: each part of the sentence is a separate line, so rewriting it doesn't affect the rest of the paragraph.
Right: rewriting the beginning of the sentence might reflow the entire paragraph.

https://rhodesmill.org/brandon/2012/one-sentence-per-line/

**Comment by bartlomieju** [comment]
This was addressed in #11857 and can be configured using `--options-text-wrap` or `fmt::textWrap` options in config file.
